Warning with Clang 11: `LOG(ERROR)` in PMIxCommands.h

Debian-10 + Clang 11, build using spack and a modified fairmq recipe to set `DDS_PLUGIN=OFF`, `BUILD_SDK=OFF`. FairMQ 1.4.39 (so not the most recent version, which might mean, that this warning has since been fixed).

```console
spack-src/fairmq/plugins/PMIx/PMIxCommands.h:273:17: warning: 'ERROR' is deprecated: Use 'error' instead [-Wdeprecated-declarations]
            LOG(ERROR) << "ERROR";
                ^
```

--- a/fairmq/plugins/PMIx/PMIx.hpp
+++ b/fairmq/plugins/PMIx/PMIx.hpp
@@ -18,6 +18,7 @@
 #include <pmix.h>
 #include <sstream>
 #include <stdexcept>
+#include <string>
 #include <type_traits>
 #include <utility>
 #include <vector>
@@ -86,7 +87,7 @@ struct value : pmix_value_t
         PMIX_VALUE_XFER(rc, lhs, static_cast<pmix_value_t*>(const_cast<value*>(&rhs)));
 
         if (rc != PMIX_SUCCESS) {
-            throw runtime_error("pmix::value copy ctor failed: rc=" + rc);
+            throw runtime_error("pmix::value copy ctor failed: rc=" + std::to_string(rc));
         }
     }
 

--- a/fairmq/plugins/PMIx/PMIxCommands.h
+++ b/fairmq/plugins/PMIx/PMIxCommands.h
@@ -270,7 +270,7 @@ class Commands
                 // LOG(trace) << "suppressing message from itself";
             }
         } else {
-            LOG(ERROR) << "ERROR";
+            LOG(error) << "ERROR";
         }
 
         if (cbfunc != nullptr) {

--- a/fairmq/plugins/PMIx/PMIxPlugin.cxx
+++ b/fairmq/plugins/PMIx/PMIxPlugin.cxx
@@ -252,7 +252,7 @@ auto PMIxPlugin::Fence() -> void
 
 auto PMIxPlugin::Fence(const std::string& label) -> void
 {
-    Fence(label);
+    Fence();
     LOG(debug) << PMIxClient() << "pmix::fence() [" << label << "] OK";
 }
 

